### DISCUSSION TITLE
loaclStorage에 저장하는 이유 ) 로그인이나 회원가입할 때 관심분야나 핸드폰번호를 왜 localstorage에 넣는지  정리

### DISCUSSION THREAD

**Discussion by Taewook1212** [FRONTENDSCHOOL8/popcorn-euid#155] [Ideas]
로그인이나 회원가입할 때 관심분야나 핸드폰번호를 왜 localstorage에 넣는지  정리해보면

1. 로그인한 유저가 각 화면마다 해당 유저의 정보에 따라 랜더링 되야할 정보를 찾을 데이터를 localstorage에서 찾으면(getStorage()이용해서?) 서버를 호출할 필요없이 (속도개선)  사용할 수 있다

2. 그래서 회원가입할 때는 데이터서버에 저장시킬건 저장하고 가입한 유저의 localStorage에 넣습니다

3. 로그인 할 때는 이미 가입할 때 저장된 서버에서 정보를 가져와서 필수기능인 관심분야나 폰번호를 localStorage에 넣습니다.

4. 이 상태로 각 유저가 해당하는 화면에 랜더링 할 때 서버에서 불러오지않고 localStorage 에서 받아 올 거는 받아오면 속도개선에 도움이 되지 않을까 합니다.

**Comment by iamjinpark** [comment]
유용한 정보 감사합니다!
